yew/src/agent/worker/queue.rs

```diff
@@ -0,0 +1,48 @@
+use std::any::TypeId;
+use std::cell::{RefCell, RefMut};
+use std::collections::{hash_map, HashMap, HashSet};
+
+/// Thread-local instance used to queue worker messages
+pub struct Queue {
+    loaded_agents: RefCell<HashSet<TypeId>>,
+    msgs_queue: RefCell<HashMap<TypeId, Vec<Vec<u8>>>>,
+}
+
+impl Queue {
+    pub fn new() -> Queue {
+        Queue {
+            loaded_agents: RefCell::new(HashSet::new()),
+            msgs_queue: RefCell::new(HashMap::new()),
+        }
+    }
+
+    pub fn borrow_msg_queue_mut(&self) -> RefMut<'_, HashMap<TypeId, Vec<Vec<u8>>>> {
+        self.msgs_queue.borrow_mut()
+    }
```

Comment:
Could this be done directly? (i.e. type `ident.msgs_queue.borrow_mut()` every time instead of calling the function).